hashing-bouncy/src/main/scala/tsec/hashing/bouncy/hashes.scala

```diff
@@ -0,0 +1,61 @@
+package tsec.hashing.bouncy
+
+sealed trait Keccak224
+
+object Keccak224 extends AsBouncyCryptoHash[Keccak224]("SHA3-224")
+
+sealed trait Keccak256
+
+object Keccak256 extends AsBouncyCryptoHash[Keccak256]("SHA3-256")
+
+sealed trait Keccak384
+
+object Keccak384 extends AsBouncyCryptoHash[Keccak384]("SHA3-384")
+
+sealed trait Keccak512
+
+object Keccak512 extends AsBouncyCryptoHash[Keccak512]("SHA3-512")
+
+sealed trait Whirlpool
+
+object Whirlpool extends AsBouncyCryptoHash[Whirlpool]("Whirlpool")
+
+sealed trait RipeMD128
+
+object RipeMD128 extends AsBouncyCryptoHash[RipeMD128]("RipeMD128")
+
+sealed trait RipeMD160
+
+object RipeMD160 extends AsBouncyCryptoHash[RipeMD160]("RipeMD160")
+
+sealed trait RipeMD256
+
+object RipeMD256 extends AsBouncyCryptoHash[RipeMD256]("RipeMD256")
+
+sealed trait RipeMD320
+
+object RipeMD320 extends AsBouncyCryptoHash[RipeMD320]("RipeMD320")
+
+sealed trait Streebog256
+
+object Streebog256 extends AsBouncyCryptoHash[Streebog256]("GOST3411-2012-256")
+
+sealed trait Streebog512
+
+object Streebog512 extends AsBouncyCryptoHash[Streebog512]("GOST3411-2012-512")
+
+sealed trait Blake2B_160
```

Comment:
maybe its better to drop the `_`

Reply:
What do you guys think?

@hrhino @ChristopherDavenport.

I added the underscore in this lonely case because reading `Blake2b160` seemed weird in my  mind. I don't think I have a single underscore anywhere else in the project.

Also, I would rather an underscore than ```Blake2b-160``` in backticks.

If you all hate it I will drop the underscores.

Reply:
Backticks are bad too IMO. Bouncy says `Blake2b160` and I think I like that a _little_ better. Maybe `Blake2B160` is a compromise.

All in all 🤷.

Reply:
Alright it only takes at least two of you to convince me. `Blake2b160` it is.